### PR TITLE
RHIDP-2063: Remove not recommended line from docs

### PR DESCRIPTION
**Pull request by abrennan89**
**Versions:**
1.1+

**Jira link:**
- https://issues.redhat.com/browse/RHIDP-2063

**Preview link:**
https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-253/admin-rhdh/#proc-install-rhdh-airgapped-environment-ocp-helm_admin-rhdh

**Reviews:**
- [x] SME: @coreydaley 
- QE: Not required, just removing a line, doesn't really change the content of the docs.
- [x] Peer review: @linfraze

### CODE REVIEW

File: modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc [hunk 1]
```diff
@@ -82,11 +82,9 @@ podman push --remove-signatures registry.redhat.io/rhel9/postgresql-15:latest de
 +
 For more information about pushing images directly to the {ocp-short} image registry, see https://access.redhat.com/solutions/6959306[How do I push an Image directly into the OpenShift 4 registry].
 +
-[TIP]
+[IMPORTANT]
 ====
-If an x509 error occurs, ensure you install the CA certificate used for {ocp-short} routes on your system.
-
-Optionally, you can append `--tls-verify=false` to the `podman` push command, note that this approach is not recommended.
+If an x509 error occurs, verify that you have installed the CA certificate used for {ocp-short} routes on your system.
```
Comment (rm3l) [inline modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc:87]: Not strictly in the scope of this PR, but I was wondering if we could probably add a link to a RH doc explaining how to install this CA certificate.
WDYT? Found this doc: https://access.redhat.com/solutions/6088891

**Review by linfraze — DISMISSED** [review]
/lgtm

File: modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc:87]: I think that would be a good idea actually - unfortunately I am having some issues accessing the customer portal today, but I will take a look at the article and OCP docs and see if there are some good links we can include 🙂

**Review by linfraze — APPROVED** [review]

**Pull request merged**